plugins/paste.md

```diff
@@ -24,7 +24,7 @@ tinymce.init({
   selector: "textarea",  // change this value according to your HTML
   plugins: "paste",
   menubar: "edit",
-  toolbar: "paste"
+  toolbar: "paste pastetext" // paste corresponds to the `Paste` menu button, pastetext corresponds to the `Paste as text` menu button
```

Comment:
The term `menu button` here is incorrect and actually means something else entirely. This should probably be `toolbar button` instead.

Reply:
I was referring to that the toolbar button is the same as the command in the menu, but the comment apparently could be interpreted in several ways.
Maybe I should just remove the comment altogether, the toolbar button names are pretty self explainable.
The main reason for the documentation update was to make it easy do discover the pastetext toolbar button as I had to dig a bit to find that there was such a button.

Reply:
> Maybe I should just remove the comment altogether, the toolbar button names are pretty self explainable.

Might be the easiest option :)

Reply:
Done, 
It's could be one my smallest PR ever :-)